src/Main.java

```diff
@@ -0,0 +1,96 @@
+        belarusianSalad.printSaladInfo();
+        belarusianSalad.addIngredientToSalad(corn);
+        belarusianSalad.printSaladInfo();
+
+        belarusianSalad.removeIngredientFromSalad("broccoli");
+        belarusianSalad.printSaladInfo();
+
+        belarusianSalad.sortVegetablesInSalad();
+        belarusianSalad.printSaladInfo();
+
+        belarusianSalad.findAndPrintIngredientByNutrition(20, 40, 7);
+
+//        // Task 2
+        ArrayList<Student> groupOfStudents = new ArrayList<>();
+        groupOfStudents.add(new Student(12, "Tatsiana", "Masiukevich", 31));
+        groupOfStudents.add(new Student(124, "Svetlana", "Mironova", 18));
+        groupOfStudents.add(new Student(125, "Marina", "Korolenko", 16));
+        groupOfStudents.add(new Student(126, "Stepan", "Volski", 18));
+        groupOfStudents.add(new Student(5, "Yan", "Tester", 28));
+        groupOfStudents.add(new Student(101, "Yana", "Zazulinskaya", 25));
+        groupOfStudents.add(new Student(129, "Dmitry", "Kulevich", 22));
+        groupOfStudents.add(new Student(99, "Pavel", "Sobchanka", 32));
+        groupOfStudents.add(new Student(100, "Vlad", "Lobakh", 25));
+        groupOfStudents.add(new Student(122, "Tatsiana", "Tester", 42));
+
+        System.out.println("-----Sort students-----");
+        StudentNoStream.sortStudentsByAge(groupOfStudents);
+        StudentStream.printStudents(groupOfStudents);
+
+        System.out.println("-----Filter students who has T in surname start-----");
+        ArrayList<Student> requiredStudents = StudentNoStream.findStudentsByFirstLetterOfSurname(groupOfStudents, 'T');
+        StudentNoStream.printStudents(requiredStudents);
+
+        System.out.println("-----Average student age is " + StudentNoStream.getAverageAge(groupOfStudents));
+
+        System.out.println("-----Map is created from the list and filtered-----");
+        StudentNoStream.createMapOfStudents(groupOfStudents);
+        HashMap<Integer, Student> filteredMapOfStudents = StudentNoStream.filterStudentsInMapById(StudentNoStream.createMapOfStudents(groupOfStudents));
+        StudentNoStream.printStudentMap(filteredMapOfStudents);
+
+//        // Task 3
+        System.out.println("-----Sort students-----");
+        ArrayList<Student> sortedStudentList = StudentStream.sortStudentsByAge(groupOfStudents);
+        StudentStream.printStudents(sortedStudentList);
+
+        System.out.println("-----Filter students who has T in surname start-----");
+        ArrayList<Student> filteredStudentList = StudentStream.findStudentsByFirstLetterOfSurname(groupOfStudents, 'T');
+        StudentStream.printStudents(filteredStudentList);
+
+        System.out.println("-----Average student age is " + StudentStream.getAverageAge(groupOfStudents));
+
+        System.out.println("-----Map is created from the list-----");
+        Map<Integer, Student> mapOfStudents = StudentStream.createMapOfStudents(groupOfStudents);
+        StudentStream.printStudentMap(mapOfStudents);
```

Comment:
Check out how Student object is printed. Object address is printed instead of name, surname, etc.